docs/make.jl

```diff
@@ -28,6 +28,16 @@ design_mmd = init_mermaid * design
 open(joinpath(dirname(@__FILE__), "src/design_mmd.md"), "w") do io
     write(io, design_mmd)
 end
+# end required for mmd
+
+# Literate for tutorials
+
+Literate.markdown(
```

Comment:
I wonder if we want to keep the tutorial generated by Literate in /docs? I see that Agents.jl take a similar approach, but their repo does not contain the generated files (they do not delete them either, just run the build step outside)

Reply:
No effect except for simpler diffs.

Reply:
Thanks @thevolatilebit, happy to do whatever changes to keep the repo cleaner! Do you mean just writing the ouput to src/anderson.md? I am looking at this line of Agents.jl https://github.com/JuliaDynamics/Agents.jl/blob/main/docs/make.jl#L29

Reply:
I cannot understand how Agents.jl keeps the repo so clean, it looks like they do not do any cleaning up within the make.jl file after making all those .md files using `Literate.markdown`. It does seem like they put them into a folder that is included in .gitignore (https://github.com/JuliaDynamics/Agents.jl/blob/main/.gitignore), and the file directly added to src/ is also in the .gitignore (agents_visualizations.md).

Reply:
Anyways, I defer to your suggestion on how we should keep the repo cleaner, either by deleting generated files at the end of make.jl or putting them into a folder that is gitignore-d.

Reply:
Yes, it definitely  does not matter now. But in parallel with removing `design_mmd.md`, we could delete the generated file using something like

```julia
rm(joinpath(@__DIR__, "src/sketches/stochastic_simulation/anderson.md"))
```

Reply:
Thanks!